source/install/install-kubernetes-operator.rst

```diff
@@ -4,7 +4,7 @@ Installing the Operators
 ============================
 
 The required operators are installed using kubectl and each operator is created in its own namespace. Shorter namespace names
-are recommended. You can install and run multiple Mattermost installations in the same cluster using different namespaces.
+are recommended. You can install and run multiple Mattermost installations in the same cluster using different namespaces. Make sure to create operators in unused namespaces only.
```

Comment:
Did you intend to make this change here or in another PR?

Reply:
no, this belongs to a different PR. Let me find out how this could happen.

Reply:
think I solved it